doc/api-examples/rust/enable-unit.rs

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT-0
+
+use clap::Parser;
+use dbus::blocking::Connection;
+use dbus::Path;
+use std::time::Duration;
+
+#[derive(Parser)]
+struct Cli {
+    /// The node name to list the units for
+    #[clap(short, long)]
+    node_name: String,
+
+    /// The names of the units to enable. Names are separated by ','.
+    #[clap(short, long, value_delimiter = ',')]
+    unit_names: Vec<String>,
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let args = Cli::parse();
+
+    let conn = Connection::new_system()?;
+
+    let bluechi_proxy = conn.with_proxy(
```

Comment:
Overall in all the examples, I would call this variable "bluechi". Yes, it is a proxy, but i'm not sure focusing on that is very helpful.